tests/test_state/test_prosumption_interface.py

```diff
@@ -0,0 +1,110 @@
+# pylint: disable=protected-access
+from unittest.mock import patch, MagicMock
+from typing import Dict
+import pytest
+from pendulum import now, DateTime
+
+from gsy_e.models.state import ProsumptionInterface
+
+
+class ProsumptionInterfaceHelper(ProsumptionInterface):
+    """Helper class to override unimplemented StateInterface's
+       abstract methods on ProsumptionInterface class."""
+
+    def get_state(self) -> Dict:
+        raise NotImplementedError
+
+    def restore_state(self, state_dict: Dict):
+        raise NotImplementedError
+
+    def delete_past_state_values(self, current_time_slot: DateTime):
+        raise NotImplementedError
+
+
+class TestProsumptionInterface:
```

Comment:
By the way, we have a `TestProsumptionInterface` residing in `tests/strategies/test_strategy_state.py`

Reply:
Thanks for the tip. I added a couple of them just to not bypass it. Also removed the other file.